### ISSUE TITLE
BUG: Angular peer dependency ^14.0.1 on ngx-popperjs@15.0.0

### ISSUE THREAD

**Issue by JakobMe** [tonysamperi/ngx-popperjs#35]
Version 15.0.0 of ngx-popperjs still has Angular ^14.0.1 as a peer dependency:

```
npm ERR! While resolving: ngx-popperjs@15.0.0
npm ERR! Found: @angular/common@15.0.3
npm ERR! node_modules/@angular/common
npm ERR!   @angular/common@"15.0.3" from the root project
npm ERR!   peer @angular/common@"15.0.3" from @angular/forms@15.0.3
npm ERR!   node_modules/@angular/forms
[...]
npm ERR! Conflicting peer dependency: @angular/common@14.2.12
npm ERR! node_modules/@angular/common
npm ERR!   peer @angular/common@"^14.0.1" from ngx-popperjs@15.0.0
npm ERR!   node_modules/ngx-popperjs
npm ERR!     ngx-popperjs@"15.0.0" from the root project
```

See `projects/ngx-popperjs/package.json`:

```json
"peerDependencies": {
    "@angular/common": "^14.0.1",
    "@angular/core": "^14.0.1",
    "rxjs": "^7.4.0"
}
```

**To Reproduce**
Use `@angular/common@15.0.3` with `ngx-popperjs@15.0.0` and try `npm install` without `--force`.

**Expected behavior**
No NPM errors caused by conflicting peer dependencies when installing the package.

**Versions**
- ngx-popperjs version 15.0.0
- @angular/common version 15.0.3
- @angular/core version 15.0.3

**Comment by tonysamperi** [comment]
@JakobMe oops my bad. I will add a test checking that versions match...
Thanks for reporting!

**Comment by tonysamperi** [comment]
Fixed in v 15.0.1. I recovered the existing tests and added one to avoid this kind of issues.
